pkg/server/server.go

```diff
@@ -49,6 +50,12 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		},
 	)
 
+	if counter, err := s.Metrics.WebhookCounter.GetMetricWithLabelValues(eventType); err != nil {
```

Comment:
what is the difference in counting webhook requests with eventType payload and counting GH events with the same eventType payload on the line 70? isn't it same?

Reply:
sorry, my bad. I have improved in https://github.com/arquillian/ike-prow-plugins/pull/142/commits/958008f0b4e9a8f83c6b002eb8fdba394a8878ba